en/developer_guide/fieldType.rst

```diff
@@ -0,0 +1,101 @@
+Field type
+==========
+
+Open Orchestra offers a RAD approach to create the Content Types.
+Further information about it are available in the `Content Types documentation`_.
+
+During creating a content type with different fields, you can choose between different types.
+By default Open Orchestra offers few types (text line, date, email, hidden, integer, choice, media, money, rich test, text area).
+
+
+.. image:: ../../images/fieldType.png
+
+Add a field type
+----------------
+To add a new field type you can add in your configuration parameters an entry
+to the ``open_orchestra_backoffice.field_types`` key
```

Comment:
you don't "add an entry to a key", you "add a <your key> entry to the configuration"